apps/src/weblab/brambleHost.js

```diff
@@ -607,7 +604,7 @@ function load(Bramble) {
     url: BRAMBLE_BASE_URL + '/index.html',
     useLocationSearch: true,
     disableUIState: true,
-    capacity: MAX_PROJECT_CAPACITY,
+    capacity: webLab_.getMaxProjectCapacity(),
```

Comment:
Why does this need to be configurable rather than the constant set here?

Reply:
a couple of reasons:
1. i wanted WebLab.js to control this limit instead of brambleHost.js
2. `<WebLabView/>` also needs access to this value as well, so i moved it into redux so that we can easily share/access it